src/GeneratorHelper.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ */
+class GeneratorHelper
+{
+    public function getFormFieldsPrintCode(array $formFields): string
+    {
+        $printCode = '';
+
+        foreach ($formFields as $fieldName) {
+            $printCode .= <<<PHP
+            ->add('$fieldName')
+PHP;
+        }
+
+        return $printCode;
+    }
+
+    public function getEntityFieldPrintCode($entity, $field): string
+    {
+        $printCode = $entity.'.'.$field['fieldName'];
+
+        if (in_array($field['type'], ['datetime'])) {
+            $printCode = $printCode.' ? '.$printCode.'|date(\'Y-m-d H:i:s\') : \'\'';
+        } elseif (in_array($field['type'], ['date'])) {
+            $printCode = $printCode.' ? '.$printCode.'|date(\'Y-m-d\') : \'\'';
+        } elseif (in_array($field['type'], ['time'])) {
+            $printCode = $printCode.' ? '.$printCode.'|date(\'H:i:s\') : \'\'';
+        } elseif (in_array($field['type'], ['array'])) {
+            $printCode = $printCode.' ? '.$printCode.'|join(\', \') : \'\'';
+        } elseif (in_array($field['type'], ['boolean'])) {
+            $printCode = $printCode.' ? \'Yes\' : \'No\'';
+        }
+
+        return $printCode;
+    }
+
+    public function getHead($baseLayoutExists, $title): string
+    {
+        if ($baseLayoutExists) {
+            $code = <<<TWIG
+{% extends 'base.html.twig' %}
+
+{% block title %}$title{% endblock %}
+
+TWIG;
+        } else {
+            $code = <<<HTML
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>$title</title>
+    </head>
+HTML;
+        }
+
+        return $code;
+    }
+
+    public function getBodyStart($baseLayoutExists): string
```

Comment:
Actually, I think we can simplify: just remove `getBodyStart` and `getBodyEnd` and *always* put `{% block body %}` and `{% endblock %}`. We already do this in `make:controller`.

The reason is that it's technically valid HTML (at least, I believe, or valid "enough") to not have a `<body>` tag. And it's also valid to have a `{% block body %}` when you are not extending anything. So let's simplify.